app/javascript/articles/components/Meta.jsx

```diff
@@ -0,0 +1,87 @@
+import { h } from 'preact';
+import {
+  articlePropTypes,
+  organizationPropType,
+} from '../../src/components/common-prop-types';
+import { PublishDate } from './PublishDate';
+
+export const Meta = ({ article, organization }) => (
+  <div className="crayons-story__meta">
+    <div className="crayons-story__author-pic">
+      {organization && !document.getElementById('organization-article-index') && (
```

Comment:
Just curious about the check for `!document.getElementById('organization-article-index')`.